presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -913,10 +918,11 @@ private Scope legacyVisitQuerySpecification(QuerySpecification node, Optional<Sc
             // to pass down to analyzeFrom
 
             Scope sourceScope = analyzeFrom(node, scope);
+            Scope sourceScopeWithWindow = createWindowScope(node, Optional.of(sourceScope));
```

Comment:
For the same reason as explained in  https://github.com/prestodb/presto/pull/8307#discussion_r124896455.